Homework5_ZhukAlexander/task3_get_ranges.py

```diff
@@ -0,0 +1,30 @@
+"""
+3. Реализовать функцию get_ranges, которая получает на вход непустой список
+неповторяющихся целых чисел, отсортированных по возрастанию, которая этот
+список “сворачивает”.
+get_ranges([0, 1, 2, 3, 4, 7, 8, 10]) // "0-4,7-8,10"
+get_ranges([4, 7, 10]) // "4,7,10"
+get_ranges([2, 3, 8, 9]) // "2-3,8-9"
+"""
+
+
+def get_ranges(enter_lst):
+    temp_list = []
+    new_list = []
+    for element in enter_lst:
+        if element + 1 not in enter_lst or element - 1 not in enter_lst:
```

Comment:
O(n**2) solution. it can be solved with O(n)

Reply:
fixed